scrapers/va/csv_bills.py

```diff
@@ -427,13 +385,5 @@ def scrape(self, session=None):
                         media_type="text/html",
                         on_duplicate="ignore",
                     )
-                    b.add_version_link(
```

Comment:
Do they not have PDFs versions available for 2025 bills? or is it no longer convenient to grab the PDF URLs based on this new data source?

Reply:
They don't include the PDF identifiers in the CSV data. I think we _could_ scrape json endpoints that power the javascript frontend (though they're using some bot detection) and pull them that way.

I'm hoping they'll be included in the API and that someone will approve my key request soon, if not then we'll have to go the endpoint route. If they don't approve my API key by next week I'll PR that, obviously feel free to do it sooner, I wouldn't be offended =).